app/src/main/java/com/vultisig/wallet/models/cosmos/THORChainAccountValue.kt

```diff
@@ -4,7 +4,7 @@ import com.google.gson.annotations.SerializedName
 
 data class THORChainAccountValue(
     val address: String?,
-    @SerializedName("account_number") val accountNumber: String?,
+    @SerializedName("account_number") val accountNumber: String,
```

Comment:
account_number can be null , when an address is not exist on the blockchain yet , then accountNumber will be null